src/fireblocks-sdk.ts

```diff
@@ -188,14 +199,60 @@ export class FireblocksSDK {
     public get NCW(): NcwSdk {
         return this.apiNcw;
     }
-
     /**
      * Get the instance of ApiClient used by the FireblocksSDK
      */
     public getApiClient(): ApiClient {
         return this.apiClient;
     }
-
+    /**
+     * Get all staking chains
+     */
+    public async getStakingChains(): Promise<string[]> {
+        return await this.stakingApiClient.getChains();
+    }
+    /**
+     * Get chain info
+     */
+    public async getStakingChainInfo(chainDescriptor: StakingChain): Promise<ChainInfo> {
+        return await this.stakingApiClient.getChainInfo(chainDescriptor);
+    }
+    /**
+     * Get staking positions summary
+     */
+    public async getStakingPositionsSummary(byVault?: boolean): Promise<DelegationSummaryDto | DelegationSummaryDtoByVault> {
+        return await this.stakingApiClient.getPositionsSummary(byVault);
+    }
+    /**
+     * Execute staking action on a chain
+     */
+    public async executeStakingAction(actionId: StakingAction, chainDescriptor: StakingChain, body: any): Promise<ExecuteActionResponse> {
```

Comment:
maybe create type for body that's stakeRequestDTO | unstakeRequestDTO | withdrawRequestDTO

Reply:
Done